__tests__/api/costumes.test.js

```diff
@@ -16,10 +16,11 @@ const request = supertest(app)
 
 const pool = getPool();
 
+afterAll(async () => {
+    await pool.end()
+})
```

Comment:
make sure this is inside a describe (maybe this was done in a later PR?)

Reply:
I think I did not wrap it in a describe on this branch because it was done on another branch that was about to merge into main. I'll make sure main has this change after all my PRs are merged.